main/ertp/guide/PursesAndPayments.md

```diff
@@ -0,0 +1,317 @@
+    const goodAmounts = Array(10).fill(quatloosAmountMath.make(10));
+    // Results in an array of 10 payments, each with value 10.
+    const arrayOfNewPayments = quatloosIssuer.splitMany(oldPayment, goodAmounts);
+    ```
+- [`issuer.isLive(payment)`](https://agoric.com/documentation/ertp/api/issuer.html#issuer-islive-payment)
+  - Returns `true` if `payment` has value. If `payment` is a promise,
+    the operation proceeds upon resolution.
+- [`mint.mintPayment(newAmount)`](https://agoric.com/documentation/ertp/api/mint.html#mint-mintpayment-newamount)
+  - Returns a new `payment` containing the newly minted assets corresponding to the `newAmount` argument. Note
+    that unlike creating a new `payment` by withdrawing existing assets from a `purse`,
+    this creates new digital assets of the specified in `newAmount` `brand`.
+  - ```js
+    const quatloos1000 = quatloosAmountMath.make(1000);
+    const newPayment = quatloosMint.mintPayment(quatloos1000);
+    ```
+- [`purse.deposit(payment, optAmount)`](https://agoric.com/documentation/ertp/api/purse.html#purse-deposit-payment-optamount)
+  - Deposit all of `payment` into this `purse`, returning the deposit
+    `amount` description. If optional `optAmount` does not equal the `payment`'s balance
+     or if `payment` is an unresolved promise, it throws an error.
+  - ```js
+    const quatloosPurse = quatloosIssuer.makeEmptyPurse();
+    const quatloosPayment = quatloosMint.mintPayment(quatloosAmountMath.make(123));
+    const quatloos123 = quatloosAmountMath.make(123);
+    // Deposit a payment for 123 quatloos into the purse. Ensure that this is what you expect.
+    quatloosPurse.deposit(quatloosPayment, quatloos123);
+    const secondPayment = quatloosint.mintPayment(quatloosmountMath.make(100));
+    // Throws error
+    quatloosPurse.deposit(secondPayment, quatloos123);
+    ```
+- [`purse.makeDepositFacet()`](https://agoric.com/documentation/ertp/api/purse.html#purse-makedepositfacet)
+  - Creates a deposit-only facet on the `purse` that can be given to
+    other parties that lets them deposit a `payment` (but not
+    withdraw) into the `purse`.
+  - ```js
+    const depositOnlyFacet = purse.makeDepositFacet();
+    // Give depositOnlyFacet to someone else. 
+    // They can pass a payment that will be deposited:
+    depositOnlyFacet.receive(payment);
+    ```
+## `purse` and `payment` example
+
+The following code creates a new `purse` for the `quatloos` brand, deposits
+10 Quatloos into the `purse`, withdraws 3 Quatloos from the `purse` into a
+`payment`, and finally returns an `amount` describing what's currently in the `purse`, 7 Quatloos.
+
+```js
+// Create a purse with a balance of 10 Quatloos
+const { issuer: quatloosIssuer, mint: quatloosMint } = makeIssuerKit('quatloos');
+const quatloosPurse = quatloosIssuer.makeEmptyPurse();
+const quatloosPayment = quatloosMint.mintPayment(quatloosAmountMath.make(10));
+const quatloos10 = quatloosAmountMath.make(10);
```

Comment:
why do we make quatloos10 twice, once in line 302 and another in line 303? Can we move line 303 up one and use it instead?

Reply:
Was trying to show how the optAmount would successfully work with it and the deposited amount being two different amount records with the same value and brand values. Is there a better/different way to do that or is it just not needed?

Reply:
we could do the following and it would shorter and more in line with how people code:

```
const quatloos10 = quatloosAmountMath.make(10);
const quatloosPayment = quatloosMint.mintPayment(quatloos10);

```